.golangci.yaml

```diff
@@ -60,3 +61,9 @@ linters-settings:
       - shadow
       - testinggoroutine
       - unusedwrite
+  depguard:
+    rules:
+      main:
+        deny:
+          - pkg: "github.com/ghodss/yaml"
+            desc: Use "sigs.k8s.io/yaml"
```

Comment:
Right! We should use kyaml instead. https://pkg.go.dev/sigs.k8s.io/kustomize/kyaml